torch_frame/data/dataset.py

```diff
@@ -63,6 +65,7 @@ class DataFrameToTensorFrameConverter:
             column name into stats. Available as :obj:`dataset.col_stats`.
         target_col (str, optional): The column used as target.
             (default: :obj:`None`)
+        sep (str): Separator for multi-categorical columns. (default: :obj:`,`)
```

Comment:
this is very weird. `sep` needs to be a stat, and it is especially column-specific.

Reply:
Can you elaborate more? Thanks~

Reply:
The seperator is not necessarily unique across columns.

Reply:
Got it.